styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java

```diff
@@ -140,7 +141,11 @@ public static void setUpClass() throws Exception {
   @AfterClass
   public static void tearDownClass() throws Exception {
     if (localDatastore != null) {
-      localDatastore.stop();
+      try {
+        localDatastore.stop(org.threeten.bp.Duration.ofSeconds(30));
```

Comment:
it's not joda-time but a library with a few backports of the new java 8 time api classes. I guess the datastore library uses it to not require java 8.